crates/rust-analyzer/src/main_loop.rs

```diff
@@ -190,7 +204,12 @@ impl GlobalState {
                     lsp_types::DocumentFilter {
                         language: None,
                         scheme: None,
-                        pattern: Some("**/rust-analyzer.toml".into()),
+                        pattern: Some("**/BUCK".into()),
+                    },
+                    lsp_types::DocumentFilter {
+                        language: None,
+                        scheme: None,
+                        pattern: Some("**/TARGETS".into()),
```

Comment:
I shouldn't be hard-coding this here; this should be coming from configuration.

(i think it's a hard blocker until this is a config, to be clear.)

Reply:
this is now driven by user-specified config.